docs/tutorials/implement-transformation-module.md

```diff
@@ -0,0 +1,7 @@
+---
+id: implement-transformation-module
+title: Implement transformation module
+sidebar_label: Implement transformation module
+---
+
+## Python API
```

Comment:
Yep, this shouldn't be a part of this PR if it's not done. Maybe it's just mistake, if yes, then delete it

Reply:
I believe this was added bcs some links were linked to this page and when deleted they couldn't build the docs, but instead of recreating the page, the links should have been fixed to link to another page... i hope I managed to fix all the links, so I'll delete it now